### PR TITLE
ZENKO-3739 Update keycloak to run ARWWI

### PR DESCRIPTION
**Pull request by ghost**
**What does this PR do, and why do we need it?**

This PR goal is to create the new CI groups and requirements in Keycloak so that Vault end2end tests can run, following the recent implementation of ARWWI.

**Which issue does this PR fix?**

fixes ZENKO-3739

**Special notes for your reviewers**:

- The PR add new keycloak users, groups and the Storage Manager role as well as mappers to make the latest Vault's ARWWI tests work as expected.
- An account suffixed with `-norights` has been added and act as the `Bartsimpson` account in Vault, i.e., an account without groups/StorageManager role, as needed in the functional tests.
- The `OIDC_PASSWORD` has been updated to be compliant with the password we use in Vault when running functional tests.
- Finally, the right role is seeded when running the tests (`storage-manager-role`).

### CODE REVIEW

File: eve/workers/end2end/scripts/configs/keycloak_config.json [hunk 1]
```diff
@@ -55,6 +237,42 @@
...
+      "consentRequired": false,
+      "config": {
+        "id.token.claim": "true",
+        "access.token.claim": "true",
+        "claim.name": "roles",
+        "multivalued": "true",
+        "userinfo.token.claim": "true"
+      }
+    }, {
+      "name": "group_matter",
```
Comment (ghost) [inline eve/workers/end2end/scripts/configs/keycloak_config.json]: groups_matter -> groups_mapper?

**Review by ghost — APPROVED** [review]
LGTM otherwise but waiting on CI fix. After discussion with William, approving

File: eve/workers/end2end/scripts/configs/keycloak_config.json (hunk 1 above)
Comment (ghost) [reply eve/workers/end2end/scripts/configs/keycloak_config.json]: done! https://github.com/scality/Zenko/pull/1408/files#diff-716fdb3e4a6824463aef5a4bcd9feda7cb0b4de2a0d2aa3cf3366886bd5cc0f9R253

**Review by XinLi0207 — APPROVED** [review]

**Pull request merged**